lib/tessel/deploy.js

```diff
@@ -68,44 +68,31 @@ Tessel.prototype.restartScript = function(opts) {
   var filepath = isPush ? PUSH_PATH : RUN_PATH;
 
   return new Promise(function(resolve, reject) {
-    return self.connection.exec(commands.readFile(filepath + opts.entryPoint))
-      .then(function(remoteProcess) {
-        var error = '';
-
-        remoteProcess.stderr.on('data', function(data) {
-          error += data.toString();
-        });
-
-        remoteProcess.once('close', function() {
-          if (error.length) {
-            if (error.indexOf('No such file or directory') !== -1) {
-              error = '"' + opts.entryPoint + '" not found on ' + self.name;
-            }
+    return self.simpleExec(commands.readFile(filepath + opts.entryPoint))
```

Comment:
Should we have a command "checkFileExistance"?
```bash
[ -f $FILE ] && echo "File exists" || echo "File does not exists"
```

Just to be super clear that we don't care about the contents of the file.